### PR TITLE
Implement Greetings Display with Message From Rails Api

### PR DESCRIPTION
**Pull request by gisachris**
## IN This PR I present the Implementation of Greetings Display From Rails Api 📡 :
Below are some of my implementations in this PR🤝 .

## Requirements
- Use git-flow 🌊.
- Create a new React app
- Create the App component with react-router.
- Create the `Greeting` component that will display a `greeting`. Set it up as a `route` in your `App` component.
- Create a `store`, an `action` and a `reducer` that will connect to you `API` endpont to get the random `greeting`.
- Display the random greeting in your `Greeting` component.

## ACHIEVEMENTS ⛰️
- Use git-flow 🌊. ✅ 
- Create a new React app. ✅ 
- Create the App component with react-router. ✅ 
- Create the `Greeting` component that will display a `greeting`. Set it up as a `route` in your `App` component. ✅ 
- Create a `store`, an `action` and a `reducer` that will connect to you `API` endpont to get the random `greeting`. ✅ 
- Display the random greeting in your `Greeting` component. ✅ 

## Conclusion
**I look forward to hearing 👂 from you The CR in terms of what can be better implemented 👨‍💻 for the project.**

Thankyou!. 🙇

### CODE REVIEW

File: src/components/message.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    };
+
+    fetchGreeting();
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h1><u>Random Greeting</u></h1>
+      <p>{greeting}</p>
+    </div>
```
Comment (elisha2kyakpo1) [inline src/components/message.js]: - [ ] Kindly troubleshoot and find out why the app can't display messages from the API. I can see some pieces of code that should be done under redux being implemented here.

messages

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @gisachris, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
👏 The API is nicely set up.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gisachris** [comment]
## IN This PR I Implemented Required Changes ♻️ :
Below are some of my implementations in this PR🤝 .

## Required Change ♻️ 
- Move fetching Functionality away from the component(`message.js`)
- Add page navigation

## ACHIEVEMENTS ⛰️
- Move fetching Functionality away from the component(`message.js`) ✅ 
- Add page navigation. ✅ 

Capture

## Conclusion
**I look forward to hearing 👂 from you The CR in terms of what can be better implemented 👨‍💻 for the project.**

Thankyou!. 🙇

**Review by Davidosky007 — APPROVED** [review]
Hi @gisachris 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* API works as expected 👍 
* Well-written PR 👍 
* Good looking readme 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**